core/src/test/java/org/infinispan/stream/BaseStreamIteratorEvictionTest.java

```diff
@@ -43,7 +44,7 @@ public void testExpiredEntryNotReturned() throws InterruptedException {
       Thread.sleep(TimeUnit.SECONDS.toMillis(expectedTime) + 50);
 
       Map<Object, String> results;
-      try (Stream<CacheEntry<Object, String>> stream = cache.getAdvancedCache().cacheEntrySet().stream()) {
+      try (CacheStream<CacheEntry<Object, String>> stream = cache.getAdvancedCache().cacheEntrySet().stream()) {
```

Comment:
Why do these need to be changed to `CacheStream` ?

Reply:
It's due to the implementation of `mapFromStream`. In there, I use now the overloaded `collect` method but that's only available in `CacheStream` API, hence the change. Anything you'd change?

Reply:
I see now, thanks.